src/services/url.ts

```diff
@@ -22,9 +22,25 @@ class UrlService extends BaseService<UrlService.Options> {
   }
 
   init() {
+    this.beautifier = new UrlBeautifier(this.generateRoutes(), this.opts.beautifier, this.app.config);
```

Comment:
can we actually move this back to the declaration of the instance variable? (line 12)